client/app/telecom/telephony/billingAccount/guides/telecom-telephony-guides.controller.js

```diff
@@ -0,0 +1,54 @@
+angular
+  .module('managerApp')
+  .controller('TelecomTelephonyGuidesCtrl', class TelecomTelephonyGuidesCtrl {
+    constructor($translate, TucToastError, TELEPHONY_GUIDES) {
+      this.$translate = $translate;
+      this.constant = { TELEPHONY_GUIDES };
+    }
+
+    $onInit() {
+      this.loading = {
+        init: false,
+      };
+      this.guides = null;
+      this.language = null;
+      this.count = null;
+
+      this.guides = this.constant.TELEPHONY_GUIDES;
+      if (localStorage['univers-selected-language']) {
+        this.language = localStorage['univers-selected-language'].replace(/-.*$|_.*$/, '');
+      } else if (navigator.language || navigator.userLanguage) {
+        this.language = (navigator.language || navigator.userLanguage).replace(/-.*$|_.*$/, '');
+      }
+      this.injectTitleInUrl();
+    }
+
+    /**
+     * Inject title in URL.
+     */
+    injectTitleInUrl() {
+      if (_.has(this.guides, 'sections')) {
+        _.forEach(this.guides.sections, (section) => {
+          if (_.has(section, 'guides')) {
+            _.forEach(section.guides, (guide) => {
+              if (_.has(guide, ['url', this.language]) && _.isString(guide.label)) {
+                guide.url[this.language] = guide.url[this.language].replace( // eslint-disable-line
+                  '{title}',
+                  _.snakeCase(this.$translate.instant(guide.label)),
+                );
+                this.count += 1;
+              }
+            });
+          }
+        });
+      }
+    }
```

Comment:
Why is this necessary ? I don't see any `${title}` to replace in given URLs ?

Reply:
I renamed the method and removed ${title}